src/shillelagh/adapters/base.py

```diff
@@ -120,6 +125,7 @@ def get_data(
         self,
         bounds: Dict[str, Filter],
         order: List[Tuple[str, RequestedOrder]],
+        **kwargs: Any,
```

Comment:
I actually think this was a breaking change for 1.1 release. I am seeing:
```
airtabledb/adapter.py:168: error: Signature of "get_data" incompatible with supertype "Adapter"
airtabledb/adapter.py:16[8](https://github.com/cancan101/airtable-db-api/runs/7884454809?check_suite_focus=true#step:9:9): note:      Superclass:
airtabledb/adapter.py:168: note:          def get_data(self, bounds: Dict[str, Filter], order: List[Tuple[str, Literal[Order.ASCENDING, Order.DESCENDING]]], **kwargs: Any) -> Iterator[Dict[str, Any]]
airtabledb/adapter.py:168: note:      Subclass:
airtabledb/adapter.py:168: note:          def get_data(self, bounds: Dict[str, Filter], order: List[Tuple[str, Literal[Order.ASCENDING, Order.DESCENDING]]]) -> Iterator[Dict[str, Any]]
Found 1 error in 1 file (checked 15 source files)
```

I can fix on my end, but just a heads up.

Reply:
Oh, thanks for letting me know! But just to confirm, this is a mypy error, right? Adapters written for 1.0.x should still work after upgrading to 1.1.x, so I wouldn't consider this a breaking change, even if this could potentially break CI/CD pipelines.

Reply:
Correct, it looks like this will just be a mypy issue (for now). At some point if this code is ever modified to always pass a kwarg, then old adapters will break:
https://github.com/betodealmeida/shillelagh/blob/6cf03406db06a994cefd16bf480935a425045125/src/shillelagh/backends/apsw/vt.py#L488-L492